### DISCUSSION TITLE
Android Auto - Map orientation while driving

### DISCUSSION THREAD

**Discussion by styrheim** [osmandapp/OsmAnd#15564] [Ideas]
If I use the map at all when planning a trip, I orient the map with North up.

Then I plug the phone into the car, start driving, and discover that North is still up. 

But I very much prefer to have movement direction _up_ while driving. Could that be the default once the car gets moving? (At least unless I specifically chose something else after plugging into the car?

Android Auto 8.2.623924-release
Osmand+ 4.2.7

**Comment by markwmuller** [comment]
This seems a related issue -- https://github.com/osmandapp/OsmAnd/issues/16015

**Comment by wehkah** [comment]
I'd like to support styrheim's request: the automatic rotation of a map should not start unless the phone is moving or the user has tapped "start". Just like him, I like to use the map with north pointing up when I plan a route.  But in the profile "car" I have set automatic rotation. This has a little disturbing side effect: when I start the app, north is up. Then I do a search for my destination, tap on "navigate", and the map will show me the calculated route with automatic rotation already activated, based on the last know movement of the phone. It is highly confusing to look at the map and see the route that you are about to take, when the map is rotated while you are not moving at all.
